components/31-molecules/checkboxes-with-filter/checkbox_filter.functions.js

```diff
@@ -242,9 +242,11 @@
     const checkboxLoop = next => {
       for (let i = checkboxes.length; i--;) {
         let checkboxWrapper = checkboxes[i];
-        let checkbox = checkboxWrapper.querySelector('input[type=checkbox]');
-        let label = checkboxWrapper.querySelector('label');
-        next({checkboxWrapper, checkbox, label});
+        if (checkboxWrapper.querySelector('input[type=checkbox]')) {
```

Comment:
Add a comment as to why the checkbox wouldn't be there.
I don't see why there would be an empty wrapper.